packages/server/safeMutations/safeArchiveTeam.ts

```diff
@@ -49,7 +49,7 @@ const safeArchiveTeam = async (teamId: string) => {
     }).run(),
     updateTeamByTeamId(updates, teamId)
   ])
-  return {...rethinkResult, users}
+  return {...rethinkResult, team: pgResult[0] ?? null, users}
```

Comment:
The `team` object that comes from RethinkDB is null if the object wasn't changed.
`RETURNING *` will always return a team object as long as it exists, so the error message in `archiveTeam` would be wrong.
This is also the only place that `RETURNING *` is used, so it feels a little bad that we're always performing an extra read just for this 1 place.

Options:
- write an `archiveTeam` query that includes `WHERE "isArchived" = false`
- query the team at the top of this function. if `isArchived == false` then return early with {team: null}. else, query the team again (maybe via dataLoader). There's a small chance of a race condition with this, but it's probably OK.
-  ???